packages/linux/package.mk

```diff
@@ -148,11 +150,16 @@ pre_make_target() {
     # copy some extra firmware to linux tree
     mkdir -p $PKG_BUILD/external-firmware
       cp -a $(get_build_dir kernel-firmware)/.copied-firmware/{amdgpu,amd-ucode,i915,radeon,e100,rtl_nic} $PKG_BUILD/external-firmware
+      cp -a $(get_build_dir intel-ucode)/intel-ucode $PKG_BUILD/external-firmware
```

Comment:
This change is unrelated to imx6.

Reply:
ah, sorry, I see that you only moved it from below. Still, why don't you move this dependency to project options?

Reply:
SDMA firmware should be built-in, I did it the same way as it is done for x86_64.

Reply:
should be or needs to be?

Reply:
I explained below.